stubs/toml/toml/decoder.pyi

```diff
@@ -1,19 +1,13 @@
-import sys
 from _typeshed import SupportsRead
-from typing import Any, Callable, Generic, MutableMapping, Pattern, Text, TypeVar, overload
+from collections.abc import Callable, MutableMapping
+from pathlib import PurePath
+from typing import Any, Generic, Pattern, TypeVar, overload
 from typing_extensions import TypeAlias
 
 _MutableMappingT = TypeVar("_MutableMappingT", bound=MutableMapping[str, Any])
+_PathLike: TypeAlias = str | bytes | PurePath
 
-if sys.version_info >= (3, 0):
-    from pathlib import PurePath
-
-    FNFError = FileNotFoundError
-    _PathLike: TypeAlias = str | bytes | PurePath
-else:
-    FNFError = IOError
-    _PathLike: TypeAlias = Text
-
+FNFError = FileNotFoundError
```

Comment:
```suggestion
FNFError: TypeAlias = FileNotFoundError
```

Reply:
This isn't a type alias, just a regular class alias, isn't it?

Reply:
I think type checkers will treat it the same either way.